project-communications/participation-communication-strategy.md

```diff
@@ -0,0 +1,131 @@
+
+## Mountain of Engagement
+
+Our Mountain of Engagement (MoE) can be seen in the below image. 
+
+[insert image here]
+
+It shows the levels of engagement and participation that there is within the Project. 
+It is important to note that there is not a hierarchy in relation to how important you are to the project. 
+Each level is crucial and fundamental to the success of the project. 
+
+The MoE allows us to map out where contributors lie and where we can provide opportunities to allow individuals to increase their participation to the next level. 
+A key difference between each level is the responsibility the individual has to the project. 
+As the level increases so does that individuals responsibility accordingly. 
+
+The MoE seeks out opportunities for contrbutors to move up the scale if they are happy to do so and the community manager will ensure this happens and opportunities are available. 
+
+## Onboarding New Members
+
+When new members show interest in being a part of the project, they will be onboarded and briefed on the project's aims and objectives as well as progress update. 
+This will be in the form of an onboarding pack which includes all materials that is needed to succesfully understand the project, as well as a 1:1 chat with the Community Manager who will ensure all information is understood. 
+
+There is also an onboarding checklist to ensure that all new members receive all the required information. 
+
+
+## In person Communications
+
+This engagement will involve face to face communication with and between contributors of the project. Due to the current COVID circumstances as well as the geographically spread out nature of this project, this will be in the form of online video conferencing (Zoom). 
+
+### Meet-ups
+
+Currently we have meet-ups scheduled every 2nd and 4th Thursday of the month, where anyone who is interested in the project can participate. 
+
+One key channel in which individuals find out about these meet-ups is through connection with Autistica and through Twitter.
+Therefore these are two engagement pathways that will be continnuously active to ensure maintenance and increase in participation. 
+
+The aim of the meet-ups is to discuss and work on different areas of the project as well as a space for onboarding. 
+
+The google doc, which provides the Agenda and a way to sign up to the session and it's breakout rooms, will be distributed to a mailing list of individuals.
+A link to the summary of notes will also be distributed after the session to ensure engagement after the session and ahead of the next one. 
+
+### Co-working (TBC) 
+
+The aim of these sessions would be for contributors to come together and work on specific topics together. 
+It will also be a space in which individuals can come together and work in collaboaration on different topics and porvide support wherever needed. 
```

Comment:
```suggestion
It will also be a space in which individuals can come together and work in collaboaration on different topics and provide support wherever needed. 
```